app/views/sidebar/_related.html.erb

```diff
@@ -17,14 +17,20 @@
   <% end %>
 
   <% if @node && (@node.has_power_tag('iso') || @node.has_power_tag('lang')) %>
-    <h4><%= t('sidebar._related.translation') %></h4>
-    <p>
-    <div id="google_translate_element"></div><script type="text/javascript">
+    <div id="google_translate_element" class="alert alert-info">
+      <script type="text/javascript">
       function googleTranslateElementInit() {
         new google.translate.TranslateElement({pageLanguage: 'en', layout: google.translate.TranslateElement.InlineLayout.SIMPLE, autoDisplay: false, multilanguagePage: true}, 'google_translate_element');
+        addLabelTranslateElement();
+      }
+      function addLabelTranslateElement() {
+        var paragraph = document.createElement('span');
+        paragraph.innerHTML = "Translation: ";
```

Comment:
Hi, this is looking good -- is there a reason the title is added using JavaScript rather than inline? I see - perhaps because the whole element is replaced? 

What about making a sub- `<div>` with the `google_translate_element` id, and putting the `<h4>` outside that, but still inside an outer `<div>` with `alert alert-info` class? That way the translate div is replaced but it's alongside the `<h4>` and still inside the `alert` box?

Thank you!